core/src/com/unciv/ui/screens/cityscreen/CityStatsTable.kt

```diff
@@ -178,6 +178,15 @@ class CityStatsTable(private val cityScreen: CityScreen): Table() {
             tableWithIcons.add("In resistance for another [${city.getFlag(CityFlags.Resistance)}] turns".toLabel()).row()
         }
 
+        val resourceTable = Table()
+        for (resourceSupply in city.getCityResources())
```

Comment:
Likewise, you want the sum here too. Otherwise, it shows each source of the resource separately